### PR TITLE
Integrating checks in applications build pipeline

### PR DESCRIPTION
**Pull request by maniSbindra**
This PR add section to show sample docker file for a containerized python application with a stage  to execute the static code analysis (pyflakes, pycodestyle and pydocstyle) against the python code.

### CODE REVIEW

File: Engineering/CodeReviews/Python.md [hunk 1]
```diff
@@ -24,5 +24,63 @@ We follow the [PEP 257](https://www.python.org/dev/peps/pep-0257/) Docstring con
 
 Flake8 is an option for teams who want a single wrapper for running Pyflakes and pycodestyle.
 
+#### Integrating automated checks in the application's build pipeline
```
Comment (seushermsft) [inline Engineering/CodeReviews/Python.md]: I love this!

Another change is proposed which adds a "Secure Coding" heading (#18), and I think to make this useful for that as well, we should create a new heading '## Running Tools' where this example will live, and then when anyone adds a new type of tools, they can update this example.

having great examples like this is what I think CSE needs so we have reusable components that dev crews can just drop into their projects.

File: Engineering/CodeReviews/Python.md [hunk 2]
```diff
@@ -24,5 +24,63 @@ We follow the [PEP 257](https://www.python.org/dev/peps/pep-0257/) Docstring con
...
+# defaults to production, compose may override this to development on build and run
+ARG APP_ENV=production
+ENV APP_ENV $APP_ENV
+
+# default to port 5000 
+ARG PORT=5000
+ENV PORT $PORT
+EXPOSE $PORT
+
+RUN useradd --user-group --create-home --shell /bin/false app &&   mkdir -p /opt/app
```
Comment (DanMass) [inline Engineering/CodeReviews/Python.md]: Why three spaces before mkdir?

**Comment by DanMass** [comment]
Please, break this out into a separate file and link to it. Ideally, we could integrate it into something like agogosml ( https://github.com/Microsoft/agogosml ) and point to that example as well.

**Comment by maniSbindra** [comment]
Closing, may reopen new PR, based on new repo structure at a later time

**Pull request closed without merging**